house/forms.py

```diff
@@ -21,5 +21,15 @@ def __init__(self, *args, **kwargs):
         self.fields['children'].required = False
 
 
-class HouseIDForm(forms.Form):
-    house_id = forms.UUIDField(label='Enter Your House ID')
+class HouseCreationForm(forms.ModelForm):
+    class Meta:
+        model = House
+        fields = (
+            'public',
+            'country',
+            'city',
+            'parent_profession_1',
+            'parent_profession_2',
+            'income',
```

Comment:
in the UI form it just says income and I think we should make this a custom field so you can alter its label to be : "Monthly Income"

Reply:
I agree 👍

Reply:
Sure, thanks